node/hyperbahn/index.js

```diff
@@ -425,3 +435,35 @@ HyperbahnClient.prototype.destroy = function destroy() {
     timers.clearTimeout(self._advertisementTimer);
     timers.clearTimeout(self.advertisementTimeoutTimer);
 };
+
+function safeSyncRead(filePath) {
+    var fileContents;
+    var error;
+
+    try {
+        fileContents = fs.readFileSync(filePath, 'utf8');
+    } catch (err) {
+        error = err;
+    }
+
+    return {
```

Comment:
you may as well use a [err, result] tuple array here, for consistency with safe-json-parse as mentioned below